app/controllers/PasskeyController.scala

```diff
@@ -143,33 +145,38 @@ class PasskeyController(
     )
   }
 
-  /** Deletes a passkey for a user */
-  def deletePasskey: Action[Map[String, Seq[String]]] = authAction(
-    parse.formUrlEncoded
-  ) { implicit request =>
-    apiResponse(
-      for {
-        passkeyId <- request.body.get("passkeyId") match {
-          case Some(values) => Success(values.head)
-          case None =>
-            Failure(
-              JanusException.missingFieldInRequest(request.user, "passkeyId")
+  /** Deletes a passkey from the user's account */
+  def deletePasskey(passkeyId: String): Action[AnyContent] = authAction {
+    implicit request =>
+      apiResponse(
+        for {
+          // Look up the passkey before deleting to include the name in the success message
+          passkeyData <- Try {
+            val queryResponse = PasskeyDB.loadCredentials(request.user).get
+            PasskeyDB
+              .extractMetadata(queryResponse)
+              .find(_.id == passkeyId)
+              .map(_.name)
+          }
```

Comment:
Would be better to pull this out of its own `Try` block:
 ```suggestion
          queryResponse <- PasskeyDB.loadCredentials(request.user)
          passkeys = PasskeyDB.extractMetadata(queryResponse)
          passkeyName <- passkeys
              .find(_.id == passkeyId)
              .map(_.name)
              .toRight(... some JanusException here ...)
              .toTry
```

Reply:
Done in 3d2401f